docs/csharp/get-started.md

```diff
@@ -28,7 +28,17 @@ Keep in mind, that this guide won't teach you C#. Instead, it teaches you how to
 
 ## Install
 
-### Installing VS Code and extensions
+### Windows
```

Comment:
I think for the official documentation we want to prioritize the common means of installing using the path of Extension Manager, etc. -- I'd prefer us show winget as an alternate method, not the default...in this context of the official vscode docs.

Reply:
agreed here